skimage/restoration/_denoise.py

```diff
@@ -1,12 +1,13 @@
 # coding: utf-8
 import numpy as np
+from math import ceil
 from .. import img_as_float
 from ..restoration._denoise_cy import _denoise_bilateral, _denoise_tv_bregman
 from .._shared.utils import _mode_deprecations
 import warnings
 
 
-def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
+def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
                       bins=10000, mode='constant', cval=0, multichannel=True):
```

Comment:
The deprecation path is as follows:

* add `sigma_range=None` to the end of the existing kwargs
* check `if sigma_range is not None`
* in the event the check succeeds, warn the user this is a deprecated kwarg which is renamed `sigma_range`
* in the event the check succeeds, check `if sigma_range != sigma_color and sigma_color is None` - if they are unequal set `sigma_color` to `sigma_range`, otherwise do nothing.
* add tests for the the new checks